termui/show_bug.go

```diff
@@ -241,7 +233,7 @@ func (sb *showBug) renderMain(g *gocui.Gui, mainView *gocui.View) error {
 	y0 += lines + 1
 
 	for i, op := range snap.Timeline {
```

Comment:
`i` is not used anymore so the compilation fail. Use `for _, op := range snap.Timeline {`